hass-addon-sunsynk-dev/run.py

```diff
@@ -54,35 +64,85 @@ async def publish_sensors(sensors: List[Filter], *, force: bool = False) -> None
 
 async def hass_discover_sensors(serial: str, rated_power: float) -> None:
     """Discover all sensors."""
-    ents: List[Entity] = []
     dev = Device(
         identifiers=[OPT.sunsynk_id],
         name=f"Sunsynk Inverter {serial}",
         model=f"{int(rated_power/1000)}kW Inverter {serial}",
         manufacturer="Sunsynk",
     )
+    # pylint: disable=import-outside-toplevel
```

Comment:
do you need this?

Reply:
Nope, can't remember where I copied and pasted it from. It's gone.